packages/graphql-language-service-server/src/startServer.js

```diff
@@ -17,67 +17,88 @@ import {
 } from './MessageProcessor';
 
 import {
+  IPCMessageReader,
   IPCMessageWriter,
   SocketMessageReader,
   SocketMessageWriter,
 } from 'vscode-jsonrpc';
 
-type ServerOptions = {
+type Options = {
   port?: number,
+  method?: string,
+  configDir?: string,
```

Comment:
Because sometimes the config dir is missing from the usage and I wanted to detect that and throw an error. I guess that could live within `index.js`.